src/bentoml/_internal/frameworks/diffusers.py

```diff
@@ -534,14 +507,14 @@ def import_model(
         if sync_with_hub_version:
             from huggingface_hub.file_download import REGEX_COMMIT_HASH
 
-            version = _extract_commit_hash(src_dir, REGEX_COMMIT_HASH)
+            version = extract_commit_hash(src_dir, REGEX_COMMIT_HASH)
             if version is not None:
                 if variant is not None:
                     version = version + "-" + variant
                 tag.version = version
 
     else:
-        _try_import_huggingface_hub()
+        is_huggingface_hub_available()
```

Comment:
Lets' then handle the exception for missing imports here.

Reply:
Got it 👍. I assume this is a follow up on the changes you suggested for `is_huggingface_hub_available`